DebugEngine/MagoNatDE/Engine.cpp

```diff
@@ -501,6 +501,8 @@ namespace Mago
         info.StdOutput = (HANDLE) hStdOutput;
         info.StdError = (HANDLE) hStdError;
         info.Suspend = true;
+		if (pszOptions && std::wstring(pszOptions) == L"--external-console")
+			info.NewConsole = true;

```

Comment:
> There is no standard flag in LAUNCH_FLAGS for external console.
> But if you prefer non-standard dwLaunchFlags over non-standard pszOptions, I can use some bit as console flag.

I think it's a bit cleaner to pass it in dwLaunchFlags as there are so many possible ways to put it into a string. Just grab a bit high enough to be unlikely to be added to the MS flags.